### PR TITLE
fix deflate and wasmbin fuzzers to easy run

### PR DESCRIPTION
**Pull request by daphneherlambda**
this PR gives the deflate and wasmbin fuzzers context to be able to reproduce them

### CODE REVIEW

**Review by Juan-M-V — APPROVED** [review]

File: wasmbin_fuzz/fuzz/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+[workspace]
+members = ["."]
+
+[[bin]]
+name = "decode"
+path = "fuzz_targets/decode.rs"
+
+[[bin]]
+name = "roundtrip"
+path = "fuzz_targets/roundtrip.rs"
```
Comment (MegaRedHand) [inline wasmbin_fuzz/fuzz/Cargo.toml]: Seeing as the file `roundtrip.rs` was deleted, should we also remove this target?

File: deflate_fuzz/fuzz/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+name = "inflate"
+path = "fuzz_targets/inflate.rs"
+test = false
+doc = false
+
+[[bin]]
+name = "libflate"
+path = "fuzz_targets/libflate.rs"
+test = false
+doc = false
```
Comment (MegaRedHand) [inline deflate_fuzz/fuzz/Cargo.toml]: I can't find these two files.

File: wasmbin_fuzz/fuzz/Cargo.toml (hunk 1 above)
Comment (daphneherlambda) [reply wasmbin_fuzz/fuzz/Cargo.toml]: yep, you´re right! deleted now

File: deflate_fuzz/fuzz/Cargo.toml (hunk 2 above)
Comment (daphneherlambda) [reply deflate_fuzz/fuzz/Cargo.toml]: those where deleted to, removed now!

**Review by MegaRedHand — APPROVED** [review]
Works on my machine 👍

**Pull request merged**